.github/actions/create-featurenet/action.yml

```diff
@@ -47,6 +47,10 @@ inputs:
     description: "Set to 'true' if it is just featurenet update"
     required: false
     default: 'false'
+  rolling-update-partition:
+    description: "Partition value when doing rolling update"
```

Comment:
Can we use this sentence from https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#partitions
> all Pods with an ordinal that is greater than or equal to the partition will be updated when the StatefulSet's .spec.template is updated. All Pods with an ordinal that is less than the partition will not be updated, and, even if they are deleted, they will be recreated at the previous version

rephrased e.g. as 

> all `aleph-node-validator-N` with an ordinal `N` that is greater than or equal to the partition will be updated. Default value: 0.

Also, this information needs to be put in Notion.

Reply:
This is now updated and the description is as you suggested

Reply:
I will ensure that Notion is updated once all this is merged